module/applications/sheets/actors/party.mjs

```diff
@@ -276,14 +280,18 @@ export default class Party extends DHBaseActorSheet {
         });
     }
 
+    get partyMembersForRoll() {
```

Comment:
Design wise I'd rather filter in the two getters you call this in or at least make this # private, but its not really a big deal..

Reply:
Moved it into the two places.